helpers.js

```diff
@@ -19,8 +19,19 @@ export function selectLanguage (language) {
 window.selectLanguage = selectLanguage;
 
 function processForm () { // eslint-disable-line no-unused-vars
-  alert('processing form (' + $('#inputtext')[0].value + ')');
-  alert('shiftLetter("a", 1) function result: ' + shiftLetter('a', 1));
+  // Get the secret key from the form.
+  var formKey = $('#key')[0].value;
+
+  // Check if formKey is a number. isNan is false if the key is
+  // empty, so let's check that too.
+  if (isNaN(formKey) || formKey === '') {
```

Comment:
Nice! Could you move this code into a separate function? You can then add some unit tests to see whether this function catches incorrect input. Thanks!

Reply:
Okay, I did it.